EMML/1-1000/EMML743.xml

```diff
@@ -45,30 +61,42 @@ schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://ra
                            <material key="parchment"/>
                         </support>
                         <extent>
-                           <measure unit="leaf"> 216+4</measure>
-                           <dimensions>
-                              <dim>30 x 26.5 cm</dim>
-                           </dimensions>
+                           <measure unit="leaf"> 216+4</measure><!-- EMIP file had 199, EMML metadata has 216 -->
```

Comment:
@Ralph-Lee-UK can you please check, I am confused by the discrepancy here between the EMIP and the HMML descriptions